docs/docs-content/clusters/data-center/nutanix/register-nutanix-cloud.md

```diff
@@ -0,0 +1,246 @@
+  | **cloudClusterTemplate.yaml**   | ConfigMap<br />Secret<br />Cluster<br />NutanixCluster<br />MachineHealthCheck | 
+  | **controlPlanePoolTemplate.yaml**| KubeadmControlPlane<br />NutanixMachineTemplate |
+  | **workerPoolTemplate.yaml**      | KubeadmConfigTemplate<br />MachineDeployment<br />NutanixMachineTemplate |
+
+
+5. In all three templates, remove all occurrences of `${NAMESPACE}`, as Palette provides its own namespace.
+
+6. In **controlPlanePoolTemplate.yaml**, edit the KubeadmControlPlane object. Rename `machineTemplate.name: ${CLUSTER_NAME}-mt-0` as `${CLUSTER_NAME}-cp-0`.
+
+7. In **controlPlanePoolTemplate.yaml**, edit the NutanixMachineTemplate object. Rename `name: ${CLUSTER_NAME}-mt-0` as `${CLUSTER_NAME}-cp-0`, and change `providerID` to `nutanix://${CLUSTER_NAME}-m1-cp-0`.
+
+ <br /> 
+
+:::caution
+The `${CLUSTER_NAME}-cp-0` parameters for the KubeadmControlPlane and NutanixMachineTemplate objects must have the same name.
+:::
+
+8. In **workerPoolTemplate.yaml**, change `providerID` to `providerID: nutanix://${CLUSTER_NAME}-m1-mt-0` within the `NutanixMachineTemplate` object. 
+    
+
+## Validate
+
+Use the steps below to confirm you have the required files and verify the required sections are removed and modified. 
+
+1. From your terminal, issue a command such as `ls -l` to list the files and confirm you have the following YAML templates:
+
+    - infrastructure-components.yaml
+    - cloudClusterTemplate.yaml    
+    - controlPlanePoolTemplate.yaml
+    - workerPoolTemplate.yaml
+
+
+2. Ensure each template contains objects as listed in the table.
+
+  | **Templates**                  | **Objects**           |
+  |--------------------------------|------------------------|
+  | **cloudClusterTemplate.yaml**    | ConfigMap<br />Secret<br />Cluster<br />NutanixCluster<br />MachineHealthCheck | 
+  | **controlPlanePoolTemplate.yaml**| KubeadmControlPlane<br />NutanixMachineTemplate |
+  | **workerPoolTemplate.yaml**     | KubeadmConfigTemplate<br />MachineDeployment<br />NutanixMachineTemplate |
+
+3. Open each file and verify that all occurrences of `${NAMESPACE}` are removed. 
+
+4. In the **controlPlanePoolTemplate.yaml** file, ensure `${CLUSTER_NAME}-cp-0` for the KubeadmControlPlane and NutanixMachineTemplate objects have the same name.
+
+5. Verify parameters are modifed as described for each template in steps 6 and 7 of [Customize YAML Configuration Files](#customize-yaml-configuration-files).
+
+## Register the Cloud
+
+Follow the steps below from your terminal to set the environment variables and invoke the APIs required to register a Nutanix cloud to Palette. Alternatively, you can use an API platform such as [Postman](https://www.postman.com/).
+
+:::caution
+During the registration process, the template files and logo file must be located in the same directory where you execute the curl commands in the steps below. The logo file must not exceed 100KB in size.
```

Comment:
And the recommend size in pixels? And ratio?

Reply:
The engineering team said that there are no requirements as of now. Only size is limited to 100KB max.

Reply:
Added: The logo file must not exceed 100KB in size. To ensure image quality ensure at least one dimension in either width or height is 40 pixels. It is preferable that the image be transparent.